src/ServerConfigParams.ts

```diff
@@ -149,6 +151,7 @@ const ConfigParamsTypes = {
     trustedVerifiers: 'string',
     relayVerifierAddress: 'string',
     deployVerifierAddress: 'string',
+    collectorAddress: 'string',
```

Comment:
Shouldn't we also rename this parameter?

Reply:
Renamed to feesReceiver

Reply:
done